app/views/subject_plans/semester_card/_add_subject_form.html.erb

```diff
@@ -17,7 +17,7 @@
     { data: { subject_selector_target: "select", action: "subject-selector#onChange" }, class: "invisible absolute" }
   ) %>
   <%= form.hidden_field :semester, value: semester %>
-  <%= form.button type: :submit, class: "cursor-pointer py-2 ps-3 material-icons text-gray-400 disabled:opacity-50 disabled:cursor-not-allowed", disabled: true, data: { subject_selector_target: "submitButton" } do %>
+  <%= form.button type: :submit, class: "cursor-pointer py-2 ps-3 material-icons text-gray-400 hover:text-green-500 disabled:hover:text-gray-400 disabled:opacity-50 disabled:cursor-not-allowed", disabled: true, data: { subject_selector_target: "submitButton" } do %>
```

Comment:
si te gusta más, creo que podés usar `hover:not-disabled:text-green-500` para cambiarle el color del hover solo si no está disabled